tests/playbooks/tests_states.yml

```diff
@@ -78,8 +78,16 @@
               - tasks/activate_profile.yml
             lsr_test:
               - tasks/remove_profile.yml
+            lsr_assert_when:
+              # Device should be absent because of NetworkManager
```

Comment:
This contradicts the user story of this test/the expected behavior: "I can remove an existing profile without taking it down" (see Thomas' comment about making the profile volatile).